app/models/pawn.rb

```diff
@@ -7,4 +7,40 @@ def unicode_symbol
       return "&#9823;"
     end
   end
+
+  def valid_move?(destination_x, destination_y)
+    valid = super(destination_x, destination_y)
+    if valid
+      if (destination_x - self.x_position).abs > 0 || (destination_y - self.y_position).abs > 1
+        valid = false
+      end
+    end
+    return valid
```

Comment:
I'm a little confused -- won't this return statement make the method end here, so matter what? How is the code below coming into play?